### PR TITLE
17166 add redirect url of path /filing

### PR DESCRIPTION
**Pull request by pwei1018**
*Issue #:* 
- bcgov/entity#17166

*Description of changes:*
- Redirect /filing to CMS URL

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://yfthig-dev--pr-140-7slv2d8v.web.app

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bc-registry",
-  "version": "2.1.0",
+  "version": "2.1.1",
```
Comment (severinbeauvais) [inline package.json:3]: Please run `npm run i` and commit package-lock.json as well.

**Review by severinbeauvais — APPROVED** [review]
LGTM

**Review by Jxio — APPROVED** [review]

**Pull request merged**